### ISSUE TITLE
Make it compatible with Qwik 1.0.0

### ISSUE THREAD

**Issue by widgeter** [qwikest/icons#8]
The dependency @qwikest/icons is not compatible with the recently released version of [Qwik 1.0.0](https://github.com/BuilderIO/qwik/releases/tag/v1.0.0)

```console
npm ERR! Could not resolve dependency:
npm ERR! peer @builder.io/qwik@">=0.19.0 <1.0.0" from @qwikest/icons@0.0.6
npm ERR! node_modules/@qwikest/icons
npm ERR!   @qwikest/icons@"^0.0.6" from the root project
```

So if a project uses **qwik** and **@qwikest/icons** at this time it cannot be upgraded to qwik 1.0.0

-> https://github.com/qwikest/icons/blob/main/package.json#L157

**Comment by widgeter** [comment]
(We are using this interesting project in [https://github.com/onwidget/qwind](https://github.com/onwidget/qwind))

**Comment by NiklasPor** [comment]
Already fixed it locally, I'll check if I'll squeeze in some other adjustments and then create a new release with `1.0.0` support

**Comment by NiklasPor** [comment]
Version `0.0.7` is released and now supports `1.0.0` of Qwik. If anyone is having any additional problems feel free to open additional issues 👍
